app/services/action_builder.rb

```diff
@@ -26,6 +26,8 @@ def member
       @user.name = @params[:name]
     end
 
+    @params[:actionkit_user_id] = AkidParser.parse(@params[:actionkit_user_id])[:actionkit_user_id] if @params.has_key? :actionkit_user_id
```

Comment:
this seems like a bit of a mixup. `AkidParser` takes an `akid` and returns a `mailing_id` and an `actionkit_user_id`, which are substrings of the dot-separated `akid`. The params then should be `params[:akid]`, no?